cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperator.java

```diff
@@ -96,31 +96,27 @@ public void start(Promise<Void> start) {
         // Configure the executor here, but it is used only in other places
         getVertx().createSharedWorkerExecutor("kubernetes-ops-pool", config.getOperationsThreadPoolSize(), TimeUnit.SECONDS.toNanos(120));
 
-        if (config.featureGates().useStrimziPodSetsEnabled()) {
-            strimziPodSetController = new StrimziPodSetController(namespace, config.getCustomResourceSelector(), resourceOperatorSupplier.kafkaOperator, resourceOperatorSupplier.strimziPodSetOperator, resourceOperatorSupplier.podOperations, config.getPodSetControllerWorkQueueSize());
-            strimziPodSetController.start();
-        }
-
         @SuppressWarnings({ "rawtypes" })
-        List<Future> watchFutures = new ArrayList<>(8);
+        List<Future> startFutures = new ArrayList<>(8);
```

Comment:
Is this 9 now?

Reply:
I count 8 futures in total, including the new one related to the PodSetController. Am I wrong?

Reply:
Ok. I did not count them, I just saw that you added additional item to it but did not changed the number. So I guess it was wrong already before maybe?

Can you please rebase it to fix the conflict and I will run the regression tests on it. Thanks

Reply:
I deployed an all ops were working fine.

Yes, I'm going to rebase.